lib/iris/fileformats/__init__.py

```diff
@@ -27,13 +27,11 @@
                                    UriProtocol, LeadingLine)
 from . import abf
 from . import um
+
 try:
-    import iris_grib as igrib
+    from . import grib as igrib
 except ImportError:
-    try:
-        from . import grib as igrib
-    except ImportError:
-        igrib = None
+    igrib = None
```

Comment:
Is there a specific reason you've chosen to import it as `igrib`? It used to be imported as just `grib`